rx/__init__.py

```diff
@@ -997,9 +1056,8 @@ def using(resource_factory: Callable[[], typing.Disposable],
 
 def with_latest_from(*sources: Observable) -> Observable:
     """Merges the specified observable sequences into one observable
-    sequence by creating a tuple only when the first
-    observable sequence produces an element. The observables can be
-    passed either as separate arguments or as a list.
+    sequence by creating a :class:`tuple` only when the first
+    observable sequence produces an element..
```

Comment:
Trailing period. Sorry to bother you :trollface:

Reply:
No bother, thanks!